src/components/CodeBlock/index.tsx

```diff
@@ -1,19 +1,62 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 import Highlight, { defaultProps } from 'prism-react-renderer'
 import { OkaidiaTheme } from '../../lib/okaidia'
+import { getCookie, generateRandomHtmlId } from '../../lib/utils'
+import './style.scss'
 
 export const CodeBlock = (props: any) => {
```

Comment:
Could these props get typed while at it?

Reply:
Done!